scripts/lib/logger/__init__.py

```diff
@@ -45,8 +45,9 @@ def format(self, record):
     prefix = '\n' if ConsoleFormatter.lineCount > 0 else ''
     if record.levelname == 'ERROR' or record.levelname == 'WARNING':
       msg = '{}[{}]: {}'.format(prefix, record.levelname, msg)
-    elif record.levelname != 'COMPLETED':
-      msg = prefix + msg
+    elif record.levelname == 'COMPLETED':
+      suffix = '\n' if ConsoleFormatter.lineCount > 1 else ''
```

Comment:
How would this work for INFO ? If we have INFO following a single COMPELTED would it be in a new line ?